pkg/apis/monitoring/v1/types.go

```diff
@@ -340,6 +340,8 @@ type CommonPrometheusFields struct {
 	// Make sure to understand the security implications if you want to enable it.
 	// When hostNetwork is enabled, this will set dnsPolicy to ClusterFirstWithHostNet automatically.
 	HostNetwork bool `json:"hostNetwork,omitempty"`
+	// GlobalPodTargetLabels are added to all Pod/ServiceMonitors' podTargetLabels
+	GlobalPodTargetLabels []string `json:"globalPodTargetLabels,omitempty"`
```

Comment:
wondering if we need the "global" prefix?

```suggestion
	PodTargetLabels []string `json:"podTargetLabels,omitempty"`
```

Reply:
I'm fine with skipping the prefix if you are :)

Reply:
With the other discussion re not allowing a monitor to drop the target labels, perhaps it would make sense to have it called `EnforcedPodTargetLabels`, to keep in line with the other policy-like configurations?

Reply:
yep this sounds good to me. As you said in the other comment, it's likely to be a global policy thing that you don't want users to by-pass and having the `enforced` prefix makes it very obvious.